packages/editor-ui/src/components/PersonalizationModal.vue

```diff
@@ -90,12 +90,12 @@
 					v-if="otherCompanyIndustryFieldVisible"
 					:value="values[OTHER_COMPANY_INDUSTRY_KEY]"
 					placeholder="Specify your company's industry"
-					@input="(value) => onInput(OTHER_COMPANY_INDUSTRY_KEY, value)"
+					@input="(value) => values[OTHER_COMPANY_INDUSTRY_KEY] = value"
 				/>
 
 
 				<n8n-input-label label="How big is your company?">
-					<n8n-select :value="values[COMPANY_SIZE_KEY]" placeholder="Select..." @change="(value) => onInput(COMPANY_SIZE_KEY, value)">
+					<n8n-select :value="values[COMPANY_SIZE_KEY]" placeholder="Select..." @change="(value) => values[COMPANY_SIZE_KEY] = value">
```

Comment:
I know this is easier than adding a helper.. but helpers are much easier to follow... it adds typing which maybe you are trying to escape here.